src/Illuminate/Support/Pluralizer.php

```diff
@@ -2,60 +2,16 @@
 
 namespace Illuminate\Support;
 
+use Illuminate\Container\Container;
 use Doctrine\Inflector\InflectorFactory;
 
 class Pluralizer
 {
     /**
      * Uncountable word forms.
      *
-     * @var string[]
+     * New uncuntable words should be added to: https://github.com/doctrine/inflector/blob/2.0.x/lib/Doctrine/Inflector/Rules/English/Uninflected.php
```

Comment:
```suggestion
     * New uncountable words should be added to: https://github.com/doctrine/inflector/blob/2.0.x/lib/Doctrine/Inflector/Rules/English/Uninflected.php
```

Reply:
Actually, as said https://github.com/doctrine/inflector/pull/194#discussion_r846107000, "related" and "recommended" are not nouns, perhaps the uncountable array and function should remain, but just for these edge cases, not nouns, nor words that already exist on the Uninflected file, with the recommendation to add new nouns to the inflector repository